app/retail/templates/shared/top_nav.html

```diff
@@ -30,11 +30,11 @@
 
 <div class="top-nav g-font-muli {{class}}">
   <a class="link d-md-block d-none" href="{% url 'products' %}">Grow Open Source</a>
-  <a class="{% if request.path|matches:'^\/((hackathons.*))$' %}selected{% endif %}" href="https://hackathons.gitcoin.co">
+  <a class="{% if request.path|matches:'^\/((hackathon.*))$' %}selected{% endif %}" href="{% url 'get_hackathons' %}">
     <img src="{% static "v2/images/top-bar/hackathons-logo.svg" %}" height="23" width="125" alt="" class="d-md-block d-none" style="margin-top: 0px; margin-left: -10px;">
     <img src="{% static "v2/images/top-bar/hackathons-symbol.svg" %}" height="22" alt="" class="d-md-none" style="height: 55px; width: 55px; margin-top: -7px; margin-left: -15px;">
   </a>
-  <a class="{% if request.path == '/' %}selected{% endif %}" href="{% url 'funder_bounties' %}">
+  <a class="{% if request.path == '/' %}selected{% endif %}" href="">
```

Comment:
hmmmm the main landing page isnt the bounties landing page..  its the main gitcoin site lander. did you mean to do this?

Reply:
good catch that was a mistake didn't mean to change nothing in that line